libs/api-test-environment/src/main/java/de/cyface/apitestutils/fixture/TestUser.java

```diff
@@ -38,7 +38,7 @@
  *
  * @author Klemens Muthmann
  * @author Armin Schnabel
- * @version 1.1.0
+ * @version 1.2.0
  * @since 1.0.0
  */
 public final class TestUser implements MongoTestData {
```

Comment:
I've implemented an alternative solution to these ugly nullable flags here. Please have a look at my commt.